desktop/Makefile

```diff
@@ -46,7 +46,7 @@ package: PACKAGE_ELECTRON_VERSION = $(shell cd $(DESKTOP_DIR) && node -e "consol
 package:
 	$(info Packaging app...)
 
-	@cd $(DESKTOP_DIR) && npx electron-builder ${ELECTRON_BUILDER_ARGS} -c.electronVersion=$(PACKAGE_ELECTRON_VERSION) build --publish never
+	@cd $(DESKTOP_DIR) && yarn electron-builder ${ELECTRON_BUILDER_ARGS} -c.electronVersion=$(PACKAGE_ELECTRON_VERSION) build --publish never
```

Comment:
We don't need `npx` as `electron-builder` is installed in `node_modules`.

I'm not sure why this works to be honest, but this comment suggests running global `electron-builder` is not the same as running it from `node_modules`: https://github.com/electron-userland/electron-builder/issues/5134#issuecomment-838093685